src/Form/FieldCheckboxMultiple.js

```diff
@@ -10,18 +10,10 @@ class FieldCheckboxMultiple extends Util.mixin(BindMixin) {
     return ['handleEvent'];
   }
 
-  handleEvent(eventType, name, modelState, event) {
+  handleEvent(eventType, name, checked, event) {
     let {props} = this;
 
-    props.handleEvent(
-      'multipleChange',
-      props.name,
-      {
-        name: name,
-        checked: modelState
-      },
-      event
-    );
+    props.handleEvent('multipleChange', props.name, [{name, checked}], event);
```

Comment:
Because radio button needs to change multiple elements in same model, e.g. if I check one, all the others should be false. This has effect on the handler, and since checkbox is using the same method, it needs to comply with the new requirements